resources/js/src/app/components/itemList/filter/ItemCategoryDropdown.vue

```diff
@@ -101,8 +99,13 @@ export default defineComponent({
   ],
 
   setup(props: CategoryDropdownProps, { root }) {
+    const pleaseSelectText = ref<string>('');
+    const categoryIsSelected = ref<boolean>(false);
```

Comment:
`categoryIsSelected` is never changed, so the variable is kind of useless. Can you just use `isCategorySelected`?

Reply:
Wrong, `categoryIsSelected` is changed during onMounted method and references `isSelected = copmuted()` which uses DOM elements and can't be parsed during server side rendering